topologies/kne/arista_ixia.textproto

```diff
@@ -2,7 +2,7 @@ name: "arista-ixia"
 nodes: {
     name: "otg"
     vendor: KEYSIGHT
-    version: "0.0.1-3423"
+    version: "0.0.1-9999"
```

Comment:
As discussed, please add some explanation (possibly a README.md in topologies/kne) about how to determine the correct version number to use.

Let's put a comment here also to indicate that this is a placeholder.

```suggestion
    # Please update this with the local version number of the foo component.
    version: "TODO"
```

Reply:
Added topologies/kne/README.md file

Reply:
Please add a comment to make the placeholder explicit.